build.sbt

```diff
@@ -88,7 +88,7 @@ lazy val mutator = project.in(file("mutator"))
 lazy val forwarder = project.in(file("forwarder"))
   .settings(Seq(
     name := "snowplow-bigquery-forwarder",
-    description := "Snowplow BigQuery Dataflow Job for replaying events from failed inserts subscription",
+    description := "This component is deprecated from version 0.5.0 on",
```

Comment:
What does the deprecation of the repeater mean in terms of BQ loader design? #74 does not give context

Reply:
Good point, I'll add a comment that Repeater should be used instead.